modules/packages/Python.chpl

```diff
@@ -2074,6 +2089,7 @@ module Python {
     */
     extern proc PySequence_Check(obj: PyObjectPtr): c_int;
     extern proc PySequence_Size(obj: PyObjectPtr): Py_ssize_t;
+    extern proc PySequence_Length(obj: PyObjectPtr): Py_ssize_t;
```

Comment:
This is probably more for my own understanding than anything else, but what's the difference between `PySequence_Size` and `PySequence_Length`?

Reply:
Absolutely nothing :smile:. https://docs.python.org/3/c-api/sequence.html#c.PySequence_Size

Reply:
Fun XD so what criteria are we using for calling one or the other?

Reply:
I don't have one, do you have a preference one way or another?

Reply:
Actually, a second look at the docs shows that `PySequence_Length` is an oddity in the docs.

- List: just size
- Set: just size
- Dict: just size
- Object: size and length
- Sequence: size and length

Makes more sense to me to use size

Reply:
Works for me, thanks!